### PR TITLE
visibility: expose ok/deny counters in admissionControl filter

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by MustafaSaber** [comment]
👍

File: filters/shedder/admission.go [hunk 1]
```diff
@@ -411,7 +411,7 @@ func (ac *admissionControl) Request(ctx filters.FilterContext) {
 	defer span.Finish()
 
 	if ac.shouldReject() {
-		ac.metrics.IncCounter(counterPrefix + ac.metricSuffix)
+		ac.metrics.IncCounter(counterPrefix + "deny." + ac.metricSuffix)
```
Comment (AlexanderYastrebov) [inline filters/shedder/admission.go:414]: Maybe `reject` to match `shouldReject`

File: filters/shedder/admission.go [hunk 2]
```diff
@@ -423,6 +423,8 @@ func (ac *admissionControl) Request(ctx filters.FilterContext) {
 		ctx.Serve(&http.Response{
 			StatusCode: http.StatusServiceUnavailable,
 		})
+	} else {
+		ac.metrics.IncCounter(counterPrefix + "ok." + ac.metricSuffix)
```
Comment (AlexanderYastrebov) [inline filters/shedder/admission.go:427]: I am not sure but maybe we should count `total` and `rejected`?

Comment (szuecs) [reply filters/shedder/admission.go]: changed

File: filters/shedder/admission.go (hunk 1 above)
Comment (szuecs) [reply filters/shedder/admission.go]: done

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**